apisix/admin/resource.lua

```diff
@@ -148,23 +233,36 @@ function _M:put(id, conf, sub_path, args)
 end
 
 
-function _M:delete(id)
+function _M:delete(id, conf, sub_path)
```

Comment:
Why do we need to pass `conf` in this method?

Reply:
@spacewander We should Keep all methods in the same form.

https://github.com/apache/apisix/blob/master/apisix/admin/init.lua#L220

The meaningful first param is id, the second one is conf, and the third one is the sub_path.

Should we?

Reply:
Let's add a comment that explains why we need a placeholder `conf` here.

Reply:
Done.